Failing `TestLoad_RunWithCALLT/loaddeployed` on Windows

Go 1.20, Windows, likely caused by improper DB/Blockchain finalisation in the test itself, firstly discovered at https://github.com/nspcc-dev/neo-go/pull/3110, https://github.com/nspcc-dev/neo-go/actions/runs/6467920919/job/17558942705?pr=3110.
```
2023-10-10T10:25:31.4419707Z === RUN   TestLoad_RunWithCALLT
2023-10-10T10:25:31.4420116Z === RUN   TestLoad_RunWithCALLT/loadgo
2023-10-10T10:25:31.4420555Z === RUN   TestLoad_RunWithCALLT/loadnef
2023-10-10T10:25:31.4420971Z === RUN   TestLoad_RunWithCALLT/loaddeployed
2023-10-10T10:25:31.4422669Z     logger.go:130: 2023-10-10T10:25:19.739Z	INFO	initial gas supply is not set or wrong, setting default value	{"InitialGASSupply": "52000000"}
2023-10-10T10:25:31.4423769Z     logger.go:130: 2023-10-10T10:25:19.739Z	INFO	mempool size is not set or wrong, setting default value	{"MemPoolSize": 50000}
2023-10-10T10:25:31.4425018Z     logger.go:130: 2023-10-10T10:25:19.739Z	INFO	P2PNotaryRequestPayloadPool size is not set or wrong, setting default value	{"P2PNotaryRequestPayloadPoolSize": 1000}
2023-10-10T10:25:31.4426163Z     logger.go:130: 2023-10-10T10:25:19.739Z	INFO	MaxBlockSize is not set or wrong, setting default value	{"MaxBlockSize": 262144}
2023-10-10T10:25:31.4427295Z     logger.go:130: 2023-10-10T10:25:19.739Z	INFO	MaxBlockSystemFee is not set or wrong, setting default value	{"MaxBlockSystemFee": 900000000000}
2023-10-10T10:25:31.4428485Z     logger.go:130: 2023-10-10T10:25:19.739Z	INFO	MaxTransactionsPerBlock is not set or wrong, using default value	{"MaxTransactionsPerBlock": 512}
2023-10-10T10:25:31.4429786Z     logger.go:130: 2023-10-10T10:25:19.739Z	INFO	MaxValidUntilBlockIncrement is not set or wrong, using default value	{"MaxValidUntilBlockIncrement": 86400}
2023-10-10T10:25:31.4430842Z     logger.go:130: 2023-10-10T10:25:19.739Z	INFO	NativeActivations are not set, using default values
2023-10-10T10:25:31.4431732Z     logger.go:130: 2023-10-10T10:25:19.739Z	INFO	Hardforks are not set, using default value
2023-10-10T10:25:31.4432619Z     logger.go:130: 2023-10-10T10:25:19.746Z	INFO	no storage version found! creating genesis block
2023-10-10T10:25:31.4433291Z     basic.go:300: contract (..\..\examples\storage\storage.go): 
2023-10-10T10:25:31.4433965Z         	Hash: 0f825b050eb8ce9eaa82993e90615025ab798016
2023-10-10T10:25:31.4436049Z         	AVM: VgIQv2AMB2RlZmF1bHRK2TAkBNswYUGb9mfOYEBXAAJYeHlTQeY/GIR4QFcAAVhZeFNB5j8YhFlAVwABWHhQQZJd6DFAWFlQQZJd6DFAVwABWHhQQS9Yxe0IQFcDAVh4EFNB3zC4mnAQwHFoQZwI7ZwmJ2hB81S/HUrZQCQE20ByaUrYJgRFwkpqEM4MATqLahHOi89xItVpQFcBAVh4EFNB3zC4mnBoQA==
2023-10-10T10:25:31.4438077Z     logger.go:130: 2023-10-10T10:25:20.895Z	DEBUG	done processing headers	{"headerIndex": 1, "blockHeight": 0, "took": "0s"}
2023-10-10T10:25:31.4439046Z     logger.go:130: 2023-10-10T10:25:20.895Z	DEBUG	done processing headers	{"headerIndex": 2, "blockHeight": 1, "took": "0s"}
2023-10-10T10:25:31.4440002Z     logger.go:130: 2023-10-10T10:25:20.895Z	DEBUG	done processing headers	{"headerIndex": 3, "blockHeight": 2, "took": "0s"}
2023-10-10T10:25:31.4441090Z     logger.go:130: 2023-10-10T10:25:20.911Z	DEBUG	done processing headers	{"headerIndex": 4, "blockHeight": 3, "took": "0s"}
2023-10-10T10:25:31.4442944Z     logger.go:130: 2023-10-10T10:25:20.975Z	INFO	persisted to disk	{"blocks": 0, "keys": 127, "headerHeight": 0, "blockHeight": 0, "took": "120.2739ms"}
2023-10-10T10:25:31.4443809Z     basic.go:300: contract (..\..\examples\runtime\runtime.go): 
2023-10-10T10:25:31.4444481Z         	Hash: 24253e67469322dd404a95e594b8b9f96445f24a
2023-10-10T10:25:31.4447944Z         	AVM: VgEMFK7Y2ztdZkygS9177TeZL/lYVgtBStkwJATbMGBB6X04oAAgKBQMC2luaXQgY2FsbGVkQc/nR5ZAVwACeSYyDCtfZGVwbG95IG1ldGhvZCBjYWxsZWQgYWZ0ZXIgY29udHJhY3QgdXBkYXRlNFZADC1fZGVwbG95IG1ldGhvZCBjYWxsZWQgYWZ0ZXIgY29udHJhY3QgY3JlYXRpb240JEBYQfgn7IwmGQwOVmVyaWZpZWQgT3duZXJBz+dHlghACUBXAAF4Qc/nR5ZAVwEBeBHAcAwFRXZlbnRoUEGVAW9hQEHpfTigACAoBAlANLRANLGqJhsMFm9ubHkgb3duZXIgY2FuIGRlc3Ryb3k6NwAAQFcAAjWM////qiYaDBVvbmx5IG93bmVyIGNhbiB1cGRhdGU6eHlQNwEAQA==
2023-10-10T10:25:31.4450831Z     logger.go:130: 2023-10-10T10:25:21.398Z	DEBUG	done processing headers	{"headerIndex": 5, "blockHeight": 4, "took": "0s"}
2023-10-10T10:25:31.4452479Z     logger.go:130: 2023-10-10T10:25:21.398Z	INFO	runtime log	{"tx": "30566a82783bda00679c708906962ad8855153a2a72c5db9c91c037cb58b642a", "script": "24253e67469322dd404a95e594b8b9f96445f24a", "msg": "init called"}
2023-10-10T10:25:31.4453996Z     logger.go:130: 2023-10-10T10:25:21.398Z	INFO	runtime log	{"tx": "30566a82783bda00679c708906962ad8855153a2a72c5db9c91c037cb58b642a", "script": "24253e67469322dd404a95e594b8b9f96445f24a", "msg": "_deploy method called after contract creation"}
2023-10-10T10:25:31.4455665Z     logger.go:130: 2023-10-10T10:25:21.434Z	INFO	persisted to disk	{"blocks": 5, "keys": 139, "headerHeight": 5, "blockHeight": 5, "took": "36.5208ms"}
2023-10-10T10:25:31.4457011Z 2023-10-10T10:25:21.680Z	INFO	runtime log	{"tx": "1138bdffe0e8975d15e7e07d96d1a65f2093a0f09a24ffd81da7c318438abc22", "script": "24253e67469322dd404a95e594b8b9f96445f24a", "msg": "init called"}
2023-10-10T10:25:31.4458248Z 2023-10-10T10:25:21.680Z	INFO	runtime log	{"tx": "1138bdffe0e8975d15e7e07d96d1a65f2093a0f09a24ffd81da7c318438abc22", "script": "24253e67469322dd404a95e594b8b9f96445f24a", "msg": "Verified Owner"}
2023-10-10T10:25:31.4459658Z     testing.go:1206: TempDir RemoveAll cleanup: remove C:\Users\RUNNER~1\AppData\Local\Temp\TestLoad_RunWithCALLTloaddeployed2213489902\001\000002.ldb: The process cannot access the file because it is being used by another process.
2023-10-10T10:25:31.4460812Z --- FAIL: TestLoad_RunWithCALLT (5.20s)
2023-10-10T10:25:31.4461858Z     --- PASS: TestLoad_RunWithCALLT/loadgo (0.56s)
2023-10-10T10:25:31.4462894Z     --- PASS: TestLoad_RunWithCALLT/loadnef (0.66s)
2023-10-10T10:25:31.4463526Z     --- FAIL: TestLoad_RunWithCALLT/loaddeployed (3.98s)
```

--- a/cli/vm/cli_test.go
+++ b/cli/vm/cli_test.go
@@ -681,6 +681,7 @@ func TestLoad_RunWithCALLT(t *testing.T) {
 		e.runProg(t,
 			"loaddeployed "+cH.StringLE()+" -- NbrUYaZgyhSkNoRo9ugRyEMdUZxrhkNaWB:Global", // the contract's owner got from the contract's code.
 			"run destroy",
+			"exit",
 		)
 		e.checkNextLine(t, "READY: loaded \\d* instructions")
 		e.checkStack(t) // Nothing on stack, successful execution.

